globalConfig.json

```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:57201/7b1651a9-dd9c-4489-949f-b5b37f7dbc71?"}
```

Comment:
We should have a newline, but that can be added later

Reply:
this is supposed to be git ignored..... I had thought I removed it